contrib/qatlib-cmake/include/mqueue.h

```diff
@@ -0,0 +1,41 @@
+#ifndef _MQUEUE_H
```

Comment:
Is that a leftover, from where is this file referenced?

Reply:
when build the target: _osal, there is a conflict issue:
1. __GLIBC_PREREQ (referenced in OsalServices.c) is only defined in `./sysroot/linux-x86_64/include/features.h`
2. mqueue.h only exist under `./sysroot/linux-x86_64-musl/`
This cause `target_include_directories` has a conflict between `./sysroot/linux-x86_64/include` and `./sysroot/linux-x86_64-musl/`, hence i decide to create mqueue.h separately under `./qatlib-cmake/include` instead of `./sysroot/linux-x86_64-musl/`.
Since the content of mqueue.h is not really reference by osal target, i clear the content and keep it blank.